content/docs/reference/clouds/_index.md

```diff
@@ -1,8 +1,10 @@
 ---
-title: Supported Clouds
+title: Supported Clouds & Languages
```

Comment:
I think at least for now - we should keep these separate. They are quite different dimensions of our product. Feels reasonable to keep them distinct at least until we want to do a broader revamp of the information architecture here.